packages/react-native-bridge/index.js

```diff
@@ -162,6 +162,16 @@ export function subscribeReplaceBlock( callback ) {
 	return gutenbergBridgeEvents.addListener( 'replaceBlock', callback );
 }
 
+/**
+ * Subscribe a listener for handling requests to open the editor help topics page.
+ *
+ * @param {Function} callback RN Callback function to display the editor
+ * 							  help topics.
+ */
+export function subscribeShowEditorHelp( callback ) {
+	return gutenbergBridgeEvents.addListener( 'showEditorHelp', callback );
```

Comment:
We will need to go ahead and stub out [the iOS version](https://reactnative.dev/docs/native-modules-ios#sending-events-to-javascript) of this event subscription bridge methods found in `react-native-bridge`. I tested running the Gutenberg Demo app on iOS and it currently launches with a failure due the missing bridge methods. 

<details><summary>Error Screenshot</summary>

<img width="401" alt="ios-demo-app-error" src="https://user-images.githubusercontent.com/438664/124184380-1a303200-da7f-11eb-8d00-10e30fc3093b.png">

</details>

Additionally, we I believe we need to open a sibling WPiOS PR as well. I think its likely that the missing bridge methods need to be implemented there at the same time. I.e. we cannot merge Android-only work if it involves bridge methods. Both platforms must be completed (or at least scaffolded in a safe way) at the same time to avoid errors like this. 

Is this something you feel comfortable implementing or would you prefer some help? I can likely contribute some time tomorrow or we could seek out an experienced iOS developer. WDYT?

Reply:
> We will need to go ahead and stub out the iOS version of this event subscription bridge methods found in react-native-bridge. I tested running the Gutenberg Demo app on iOS and it currently launches with a failure due the missing bridge methods.

Fixed in a2ba1465364896e01759b1d656e7367b7e09977e (or at least I'm pretty sure it is since the iOS demo app is running without issue now) 👍 

> Additionally, we I believe we need to open a sibling WPiOS PR as well. [...] Is this something you feel comfortable implementing or would you prefer some help? I can likely contribute some time tomorrow or we could seek out an experienced iOS developer. WDYT?

Very good point. I'll see if I can dig up something tonight and if not, I'd definitely appreciate your help tomorrow. I'd like to get further in the bottom sheet help before I'm pulled off to start my next project. If you don't have time tomorrow then I'm happy to continue cracking at it.

Reply:
> I'll see if I can dig up something tonight and if not, I'd definitely appreciate your help tomorrow. I'd like to get further in the bottom sheet help before I'm pulled off to start my next project. If you don't have time tomorrow then I'm happy to continue cracking at it.

If you feel it is most effective for me to explore the WPiOS while you look at other pieces, I can switch over to this tomorrow and see what I can find. Just let me know if you change your mind or make any progress on it otherwise. I will plan to investigate tomorrow morning. 

Regardless, I _greatly_ appreciate the time you are putting into this project. Thank you.

Reply:
Thanks David! I finished [the WPiOS implementation](https://github.com/wordpress-mobile/WordPress-iOS/pull/16795). Another great learning experience! 🎉